app/src/main/java/io/github/muntashirakon/setedit/Utils/Shortcut/ShortcutEditItemModel.java

```diff
@@ -0,0 +1,20 @@
+package io.github.muntashirakon.setedit.Utils.Shortcut;
+
+public class ShortcutEditItemModel {
+    private final String Name;
+    private final String Detail;
```

Comment:
Class fields begins with a small letter, not capital letters.

Reply:
Fixed